tests/test_data_collator_completion_only.py

```diff
@@ -40,3 +41,29 @@ def test_data_collator_finds_response_template_llama2_tokenizer(self):
         # Pass already tokenized (w context) and truncated response_template so token_ids are like in the instruction + response
         self.collator = DataCollatorForCompletionOnlyLM(self.tokenized_response_w_context, tokenizer=self.tokenizer)
         self.collator.torch_call([self.tokenized_instruction])
+
+    def test_data_collator_handling_of_long_sequences(self):
+        self.tokenizer = AutoTokenizer.from_pretrained("upstage/Llama-2-70b-instruct-v2")
```

Comment:
Can we change this to
```suggestion
        self.tokenizer = AutoTokenizer.from_pretrained("trl-internal-testing/dummy-GPT2-correct-vocab")
```
So we don't depend on external repositories. Could you also change it in the test above, it seems that slipped review when it was added.